todo_app/app.py

```diff
@@ -4,46 +4,50 @@
 import todo_app.view_model as view_model
 from todo_app.Task import Task
 import todo_app.trello as trello
+from todo_app.mongo_db_tasks import TasksDb
 
 app = Flask(__name__)
 headers = {
    "Accept": "application/json"
 }
 
-def create_app():
-    app = Flask(__name__)
-    
+taskDb = TasksDb()
 
+def create_app(taskDb):
```

Comment:
This is a nice structure for mocking tests, but worth noting that it now means this file can't be run directly with `poetry run flask run`- that's not a problem but we should update the default FLASK_ENV in your `.env.template` file to be `todo_app/wsgi` rather than `todo_app/app` since the latter will fail